pkg/resources/machinecontroller/deployment.go

```diff
@@ -53,7 +53,7 @@ var (
 const (
 	Name = "machine-controller"
 
-	tag = "v1.14.3"
+	tag = "v1.15.0"
```

Comment:
We should consider using 1.15.1 instead.

Reply:
Although, it seems like the change in 1.15.1 doesn't really affect Kubermatic, so I'll leave it up to you to choose the version.